packages/@react-aria/combobox/src/useComboBox.ts

```diff
@@ -79,7 +79,8 @@ export function useComboBox<T>(props: AriaComboBoxProps<T>, state: ComboBoxState
     keyboardDelegate: layout,
     shouldTypeAhead: false,
     disallowEmptySelection: true,
-    disallowSelectAll: true
+    disallowSelectAll: true,
+    ref: inputRef
```

Comment:
Adding this makes it so the user can no longer tab away from the combobox (bug). This happens for ActionGroup as well, will handle it here: https://jira.corp.adobe.com/browse/RSP-1771